### PR TITLE
Update dcos-aws-cli example with new command.

### PR DESCRIPTION
**Pull request by joerg84**

### CODE REVIEW

File: dcos-aws-cli/1.9/README.md [hunk 1]
```diff
@@ -21,12 +21,22 @@ $ dcos package install dcos-aws-cli
 
 ## Usage
 
-The DC/OS AWS CLI currently supports only one command:
+The DC/OS AWS CLI currently supports only two command:
```
Comment (mattj-io) [inline dcos-aws-cli/1.9/README.md]: command should be commands

File: dcos-aws-cli/1.9/README.md [hunk 2]
```diff
@@ -21,12 +21,22 @@ $ dcos package install dcos-aws-cli
...
 ```bash
 $ dcos dcos-aws-cli publicIPs
+35.158.189.26
+```
+
+- List all applications and their ports running on public agents
+Checking all apps running on public agents and their ports might is still a tedious task. This command lists all apps which have reserved port resources on a public Agent. Note that, this does not--yet--include applications which are exposed via marathon-lb or applications which use unreserved ports.
+
+```bash
+$ dcos dcos-aws-cli exposedApps
```
Comment (mattj-io) [inline dcos-aws-cli/1.9/README.md]: One small point that applies to both of these commands is that none of our other CLI commands are camelcase. Is there a specific reason for doing it that way ?

File: dcos-aws-cli/1.9/README.md [hunk 3]
```diff
@@ -21,12 +21,22 @@ $ dcos package install dcos-aws-cli
 
 ## Usage
 
-The DC/OS AWS CLI currently supports only one command:
+The DC/OS AWS CLI currently supports only two command:
 - Listing of external IP addresses
 Retrieving the externally resolvable IP addresses of public Agents can be difficult, as DC/OS internally used the interal IP addresses. This command will list the external IP addresses for all public agents.
```
Comment (mattj-io) [inline dcos-aws-cli/1.9/README.md]: typo - should be internal

File: dcos-aws-cli/1.9/README.md [hunk 4]
```diff
@@ -21,12 +21,22 @@ $ dcos package install dcos-aws-cli
...
 
 ```bash
 $ dcos dcos-aws-cli publicIPs
+35.158.189.26
+```
+
+- List all applications and their ports running on public agents
+Checking all apps running on public agents and their ports might is still a tedious task. This command lists all apps which have reserved port resources on a public Agent. Note that, this does not--yet--include applications which are exposed via marathon-lb or applications which use unreserved ports.
```
Comment (mattj-io) [inline dcos-aws-cli/1.9/README.md]: Picky grammar point - not sure about the dashes and comma here, I would just put "Note that this does not yet include"

File: dcos-aws-cli/1.9/README.md (hunk 2 above)
Comment (joerg84) [inline dcos-aws-cli/1.9/README.md]: @mattj-io cli commands don't support exposed-apps afaik.... what other example did you see?

**Comment by joerg84** [comment]
@mattj-io PTAL

File: dcos-aws-cli/1.9/README.md [hunk 5]
```diff
@@ -21,12 +21,22 @@ $ dcos package install dcos-aws-cli
 
 ## Usage
 
-The DC/OS AWS CLI currently supports only one command:
+The DC/OS AWS CLI currently supports only two commands:
 - Listing of external IP addresses
-Retrieving the externally resolvable IP addresses of public Agents can be difficult, as DC/OS internally used the interal IP addresses. This command will list the external IP addresses for all public agents.
+Retrieving the externally resolvable IP addresses of public Agents can be difficult, as DC/OS internally used the internal IP addresses. This command will list the external IP addresses for all public agents.
```
Comment (mattj-io) [inline dcos-aws-cli/1.9/README.md]: sorry spotted another typo - used should be uses

File: dcos-aws-cli/1.9/README.md [hunk 6]
```diff
@@ -21,12 +21,22 @@ $ dcos package install dcos-aws-cli
...
 
 ```bash
 $ dcos dcos-aws-cli publicIPs
+35.158.189.26
+```
+
+- List all applications and their ports running on public agents
+Checking all apps running on public agents and their ports might is still a tedious task. This command lists all apps which have reserved port resources on a public Agent. Note that this does not--yet--include applications which are exposed via marathon-lb or applications which use unreserved ports.
```
Comment (mattj-io) [inline dcos-aws-cli/1.9/README.md]: Not sure what 'might' is doing there - think this should be 'can be a tedious task' ?

**Comment by joerg84** [comment]
@mattj-io PTAL

**Pull request merged**